pyproject.toml

```diff
@@ -552,12 +551,10 @@ module = [
     "sentry.notifications.notifications.activity.base",
     "sentry.notifications.notifications.activity.new_processing_issues",
     "sentry.notifications.notifications.activity.release",
-    "sentry.notifications.notifications.base",
```

Comment:
Hi Snigdha, I recommend separating PRs for typing changes in order to minimize the mental load of reviewing them. It makes it a bit easier to review.